File: customtext0113/src/main/java/com/pltest/customtext0113/MainActivity.java
package com.pltest.customtext0113;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final TextView tv = findViewById(R.id.tvDelete);
        tv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                tv.setText(tv.getText()+"1");
            }
        });
    }
}

File: demo0227/src/main/java/com/pltest/demo0227/MainActivity.kt
package com.pltest.demo0227

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        //        findViewById(R.id.btStartAct).setOnClickListener(new View.OnClickListener() {
//            @Override
//            public void onClick(View v) {
//                startActivity(new Intent(MainActivity.this, SecondActivity.class));
//            }
//        });
        val dog: Animal = Dog()
        val bird: Animal = Bird()
        dog.call()
        bird.call()
        (dog as Dog).run()
        (bird as Bird).fly()
    }
}